common/src/services/stateMigration.service.ts

```diff
@@ -448,6 +450,23 @@ export class StateMigrationService<
     }
   }
 
+  protected async migrateStateFrom2To3(): Promise<void> {
```

Comment:
I believe you're missing a call to this method from `migrate`

Reply:
Good catch! StateVersion.Latest also needs to be bumped up to three

Reply:
Yikes! I definitely had this in another branch, obviously lost it along the way when squashing/rebasing

Reply:
Should be fixed now, called from `migrate` and updated the `stateVersion` at the end of the migration method itself.